README.md

```diff
@@ -131,6 +131,7 @@ async main() {
     "model_list": [
       {
         "model_url": myLlamaUrl,
+        "model_lib": "llama-v1-3b-q4f32_0",
```

Comment:
we can use model_lib: to directly point to a wasm url, so remove the model_lib indirection

Reply:
That also enables us to remove the lib map indirection and focus on concept of "model" and "model library"